packages/edge-login-ui-rn/src/components/themed/AccountInfo.js

```diff
@@ -64,7 +64,7 @@ const AccountInfoComponent = ({
   const headerStyle = useAnimatedStyle(() => ({
     borderBottomLeftRadius: progress.value === 0 ? borderRadius : 0,
     borderBottomRightRadius: progress.value === 0 ? borderRadius : 0,
-    borderBottomWidth: progress.value === 0 ? theme.thinLineWidth : 0
+    borderBottomWidth: progress.value === 0 ? 0.25 : 0
```

Comment:
I think here and below we need to use `StyleSheet.hairlineWidth` because on some devices we won't be able to see such a thin line